src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/health/PaymentsQueueHealthIndicator.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.health;
+
+import com.microsoft.azure.servicebus.IMessageReceiver;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PaymentsQueueHealthIndicator implements HealthIndicator {
```

Comment:
Ideally detail of health could be called just `queue`. So I think we should rename this class to `QueueHealthIndicator` - doesn't really matter what kind of queue it checks for the health endpoint perspective